build/yamls/flow-visibility/start-flow-visibility.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Curious - any reason we use "/usr/bin/env bash", not "/usr/bin/bash" or just "bash"?

Reply:
I was just following the other scripts convention in [antrea/ci ](https://github.com/antrea-io/antrea/tree/main/ci) and [antrea/test/e2e/infra/vagrant/](https://github.com/antrea-io/antrea/tree/main/test/e2e/infra/vagrant). I suppose let `env` find `bash` making the script easier to run on different systems. I don't have a strong opinion on it.

Reply:
it's good practice to use `#!/usr/bin/env bash` as in theory bash can be installed in a different location (not `/usr/bin/`).

Reply:
Got it.